go/ms/internal/mscmn/common.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package mscmn
+
+import (
+	"net"
+
+	"github.com/scionproto/scion/go/lib/addr"
+	"github.com/scionproto/scion/go/lib/env"
+	"github.com/scionproto/scion/go/lib/infra/infraenv"
+	"github.com/scionproto/scion/go/lib/infra/messenger"
+	"github.com/scionproto/scion/go/lib/infra/modules/itopo"
+	"github.com/scionproto/scion/go/lib/serrors"
+	"github.com/scionproto/scion/go/ms/internal/msmsgr"
+	"github.com/scionproto/scion/go/ms/internal/validator"
+	msconfig "github.com/scionproto/scion/go/pkg/ms/config"
+)
+
+var (
+	IA   addr.IA
```

Comment:
They are used later in some files that are not committed yet. This was just the initial code

Reply:
I see. I tried to cross-check with https://github.com/suprajasridhara/scion/pull/2, but since I also didn't see any use there, I thought it was some left over declarations.

Reply:
Oh you are right. I was looking at a different service. The mapping service does not need this. I will remove it